File: app/src/main/java/org/michaelbel/application/rest/model/Country.java
package org.michaelbel.application.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import io.realm.RealmObject;

@SuppressWarnings("all")
public class Country extends RealmObject implements Serializable {

    @SerializedName("iso_3166_1")
    public String country;

    @SerializedName("name")
    public String name;
}

File: data/src/main/java/org/michaelbel/data/remote/model/Country.kt
package org.michaelbel.data.remote.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Country(
        @SerializedName("iso_3166_1") val country: String,
        @SerializedName("title") val name: String
): Serializable